run.go

```diff
@@ -78,6 +78,7 @@ const (
 	RunPrePlanCompleted   RunStatus = "pre_plan_completed"
 	RunPrePlanRunning     RunStatus = "pre_plan_running"
 	RunQueuing            RunStatus = "queuing"
+	RunAwaitingDecision   RunStatus = "post_plan_awaiting_decision"
```

Comment:
The name should be prefixed with the stage as overrides _can_ occur at any stage. https://github.com/hashicorp/atlas/blob/main/app/models/run.rb#L36-L41
```suggestion
	RunPostPlanAwaitingDecision   RunStatus = "post_plan_awaiting_decision"
```